src/openrct2/Limits.h

```diff
@@ -0,0 +1,30 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2022 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+#include "rct2/Limits.h"
+
+namespace OpenRCT2::Limits
+{
+    constexpr const uint16_t MaxRidesInPark = 1000;
+    constexpr const uint16_t MaxStationsPerRide = 255;
+    constexpr const uint8_t CustomerHistorySize = RCT12::Limits::CustomerHistorySize;
+    constexpr const uint16_t MaxGolfHoles = RCT12::Limits::MaxGolfHoles;
+    constexpr const uint16_t MaxHelices = RCT12::Limits::MaxHelices;
+    constexpr const uint16_t MaxInversions = RCT12::Limits::MaxInversions;
+    constexpr const uint16_t MaxTrainsPerRide = 255;
+    constexpr const uint16_t MaxCarsPerTrain = 255;
+    constexpr const uint16_t MaxVehicleColours = MaxTrainsPerRide; // this should really be MaxTrainsPerRide * MaxCarsPerTrain
```

Comment:
This one seems to be unused. Is that because `MaxTrainsPerRide` in used where it should be this `MaxVehicleColours` now?

Reply:
Yes, it would be split up again. I did have it split but that was reverted in the previous iteration of this pr